### PR TITLE
feat: add CircleCI completion spec

### PR DESCRIPTION
**Pull request by import-brain**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
feature: add circleci spec
**What is the current behavior? (You can also link to an open issue here)**
Normal
**What is the new behavior (if this is a feature change)?**
New autocomplete suggestions for the CircleCI CLI
**Additional info:**
N/A

### CODE REVIEW

File: src/circleci.ts [hunk 1]
```diff
@@ -0,0 +1,203 @@
+const completionSpec: Fig.Spec = {
+  name: "circleci",
+  description: "CircleCI CLI",
+  subcommands: [
+    {
+      name: "completion",
+      description: "Generate shell completion scripts",
+      options: [
```
Comment (QuiiBz) [inline src/circleci.ts:8]: Each `options` that doesn't start with `-` (or `--`) should be `subcommands`, could you please change all of them?

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Sorry for the delay on this PR, but thanks a lot for opening it!

File: src/circleci.ts (hunk 1 above)
Comment (import-brain) [reply src/circleci.ts]: Done

**Review by mschrage — APPROVED** [review]
Thanks for making these changes @import-brain! Really appreciate your contribution :)

**Comment by import-brain** [comment]
No problem!

**Pull request merged**